config/database.go

```diff
@@ -416,3 +416,19 @@ func (ds *DatabaseStore) String() string {
 func (ds *DatabaseStore) Close() error {
 	return ds.db.Close()
 }
+
+// removes configurations from database if they are older than treshold days.
+func (ds *DatabaseStore) cleanUp(tresholdDays int) error {
```

Comment:
Simpler to pass the exact createAt in ms from the app layer. Then the query can remain same for both DBs.

```
dur := time.Duration(*s.Config().JobSettings.CleanupConfigThresholdDays) * time.Hour * 24
expiry := model.GetMillisForTime(time.Now().Add(-dur))
```

And then `DELETE FROM Configurations Where CreateAt < ?`

Reply:
Sure, why not 👍